discovery/syncer.go

```diff
@@ -445,10 +449,8 @@ func (g *gossipSyncer) channelGraphSyncer() {
 // required to ensure they fit into a single message. We may re-renter this
 // state in the case that chunking is required.
 func (g *gossipSyncer) synchronizeChanIDs() (bool, error) {
-	// Ensure that we're able to handle queries using the specified chan
-	// ID.
-	chunkSize, ok := encodingTypeToChunkSize[g.cfg.encodingType]
-	if !ok {
+	// Ensure that we're able to handle queries using the specified chan ID.
+	if _, ok := encodingTypeToChunkSize[g.cfg.encodingType]; !ok {
```

Comment:
Can remove this check now? Not sure if it brings any value anymore.

Reply:
Yup, just ended up removing it.